### ISSUE TITLE
editing the script not possible

### ISSUE THREAD

**Issue by wicky7772** [dotnet/try#825]
So im not sure if its meant to be possible or not but when im writing codes in your tutorial i cant click on the mistaken word and fix i can only click the end on a paragraph so i have to click at the end of the line of code and back arrow key the whole line to get to the problem please make it so u can click anywhere along the line of text hope that makes sense

Example in step 2 : You may have also noticed that the word "Hello" was missing in the last two messages. Let's fix that now. Modify the lines that print the message to the following:

i cant click next to the bracket and type in new command i have to click at the end of the line of code back arrow key to the desired area add new text and if i make a mistake do same again

**Comment by Yawhou** [comment]
Yes, this issue is still there, this happens to me mainly at the last line of code,

[First thought]
- For example:
- <img width="179" alt="Screenshot 2024-06-10 at 11 49 08 PM" src="https://github.com/dotnet/try/assets/54216858/334e0885-19b6-4a7d-a49a-75fb80c3cf1f">
- The last line of the code is line31, and I now I want to change all the 'j's to 'J's. 
- From line 27-30 I can move the caret to anywhere I want with my cursor, for this case between j and e, I can do that without any problem.
- The problem is that I can't do it on line 31, regardless of where the caret is currently at, when I clicked on anywhere along line 31 the caret will jump to the end of the line.

[Second thought]
- maybe it occurs when the .net editor is out of focus and refocus again? After further trying, no it's not about focus.

[Final thought]
- I noticed it's the way how each line in the .net editor registers clicks from my cursor
- Scenario: I'm trying to highlight line 3,
- <img width="397" alt="Screenshot 2024-06-12 at 4 45 03 PM" src="https://github.com/dotnet/try/assets/54216858/ab7d973e-614f-4e52-8e34-24a01f5e459a">
- When the cursor moved to anywhere slightly below the middle part of line 3 it will consider that my cursor is now at line 4 and highlight line 4 as well.
- <img width="390" alt="Screenshot 2024-06-12 at 4 44 41 PM" src="https://github.com/dotnet/try/assets/54216858/a4957050-8509-4da2-a2ce-b953c2f537bf">
- If I do it while making sure my cursor is at the middle or above from the middle of line 3, I'm all good
- This applies to when I'm just trying to click on each line
- Tried .net editor on Firefox, Chrome, Edge, Arc, Safari, and each of them registers clicks slightly different.

[Workaround]
- Current workaround for me is to always add one extra empty line at the end of the editor (or below last line of code)
- although it doesn't solve the problem mentioned in Scenario, it does make editing on last line of code easier.
